### ISSUE TITLE
Manage all build nodes under one network

### ISSUE THREAD

**Issue by grahamc** [NixOS/nixos-org-configurations#60]
Right now there are a few groups of managed build nodes. This makes it difficult to ensure they're all running consistent versions of Nix and NixOS, and are participating in the monitoring.

 - [ ] Import the x86 Linux machines in to the network:
    - [ ] packet-epyc-1
    - [ ] packet-t2-4
    - [ ] t2a
    - [ ] t4a
    - [ ] t4b

 - [ ] Import the macOS-on-Linux machines in to the network:
    - [ ] mac2-host
    - [ ] mac3-host
    - [ ] mac6-host
    - [ ] mac7-host

 - [ ] Convert the remaining macOS-on-Darwin machines to be macOS-on-Linux and import in to the network:
    - [ ] mac1
    - [ ] mac4
    - [ ] mac5
    - [ ] mac8
    - [ ] mac9

 - [ ] Import the aarch64-linux machines in to the network:
    - [ ] add an aarch64 remote builder to the deploy host
    - [ ] packet-t2a-1
    - [ ] packet-t2a-2

**Comment by grahamc** [comment]
cc @vcunat I'll need help / configs from you for t2a/t4a/t4b

**Comment by vcunat** [comment]
Yes, I had it in my TODO after our talk.  I'll need to figure out how to integrate my custom changes &ndash; I expect to PR a suggestion.
